package.json

```diff
@@ -18,7 +18,8 @@
   "devDependencies": {
     "eslint": "8.56.0",
     "eslint-plugin-eslint-plugin": "5.3.0",
-    "eslint-plugin-node": "11.1.0"
+    "eslint-plugin-n": "17.10.2",
+    "prettier": "3.3.3"
   },
   "engines": {
```

Comment:
This review came from noticing that a typescript only lib had angular in the node_modules. Since npm made peerDependencies always be installed, installing this plugin causes angular to be included as a transitive dependency through angular-eslint.

It also brings dozens of other packages besides. Setting optional to true in peerDependenciesMeta lets us restrict the version if the package is installed, without forcing it.